src/site/content/en/reliable/app-shell-ux-with-service-workers/index.md

```diff
@@ -149,15 +149,21 @@ Next, tell Workbox how to use the strategies to construct a complete, streaming
 
 ```javascript
 workbox.streams.strategy([
-  () => cacheStrategy.makeRequest({request: '/head.html'}),
-  () => cacheStrategy.makeRequest({request: '/navbar.html'}),
+  () => cacheStrategy.handle({
```

Comment:
In v6 these will also need to pass an event object, so we may as well include that now so it doesn't have to be changed again in a few weeks.

Reply:
Good point. Though if we're thinking about this with an eye towards v6, maybe the example should be updated to use `PrecacheStrategy` directly, instead of having to configure `CacheFirst`?

Reply:
Yeah, that's a good idea, though not sure if that'll mean we have to wait to merge this until v6 comes out?

Reply:
We can just merge this now (without requiring `event`) and then file an issue to remind us to update the post once v6 is fully released.